tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -1498,8 +1498,8 @@ TEST_F(EnergyPlusFixture, DXCoil_ValidateADPFunction)
                                     DXCoil(1).RatedSHR(1),
                                     true);
 
-    EXPECT_NEAR(0.788472, DXCoil(1).RatedSHR(1), 0.0000001);
-    EXPECT_NEAR(0.0003944, CBF_calculated, 0.0000001);
+    EXPECT_NEAR(0.792472, DXCoil(1).RatedSHR(1), 0.0000001);
+    EXPECT_NEAR(0.00213735, CBF_calculated, 0.0000001);
```

Comment:
That small change on SHR caused a pretty big difference in the bypass factor.  I don't disagree, just noting.